pkgs/applications/editors/cpeditor/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, qt5
+, cmake
+, ninja
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cpeditor";
+  version = "6.9.3";
+
+  src = fetchFromGitHub {
+    owner = "cpeditor";
+    repo = "cpeditor";
+    rev = version;
+    sha256 = "sha256-yfHiTslKPRmFkSIj1WvaD4ZruL1azqjNAYbag1k2RzU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ninja pkg-config qt5.wrapQtAppsHook ];
+  buildInputs = [ qt5.qtbase qt5.qttools python3 ];
+
+  preConfigure = ''
+    sed 's,/bin/bash,${stdenv.shell},' -i src/Core/Runner.cpp
```

Comment:
Maybe use `substituteInPlace`?

Reply:
Should also be `runtimeShell` instead of stdenv.shell